auto-discovery/kubernetes/values.yaml

```
@@ -27,63 +27,67 @@ config:
     passiveReconcileInterval: 1m
     scanConfig:
```

Ah its also still wrong here
```suggestion
    scanConfigs:
```